rln/benches/circuit_loading_benchmark.rs

```diff
@@ -12,7 +12,7 @@ pub fn key_load_benchmark(c: &mut Criterion) {
 
 criterion_group! {
     name = benches;
-    config = Criterion::default().measurement_time(std::time::Duration::from_secs(250));
+    config = Criterion::default().without_plots().measurement_time(std::time::Duration::from_secs(10));
```

Comment:
need without plots, or the ci will fail with precision error on f64 values (too small time interval)

Reply:
ref: https://github.com/bheisler/criterion.rs/issues/388#issuecomment-1427653981